x-pack/plugins/ml/public/datavisualizer/selector/datavisualizer_selector.js

```diff
@@ -97,7 +96,7 @@ export function DatavisualizerSelector() {
             />
           </EuiFlexItem>
         </EuiFlexGroup>
-        {shouldShowStartTrialLink === true &&
+        {startTrialVisible === true &&
           <React.Fragment>
```

Comment:
Just a nitpick - any reason we can't just check `startTrialVisible` (without the ===) ?

Reply:
no reason really. I also often like to be explicit like this just to be clear that it's a boolean we're checking and not some other kind of truthy variable.